authkit/postgres/migrations/00003_add_user_rank.sql

```
@@ -0,0 +1,11 @@
+-- SPDX-License-Identifier: Apache-2.0
+
+-- +goose Up
+ALTER TABLE auth.users ADD COLUMN rank text NOT NULL DEFAULT '';
+
+CREATE INDEX users_rank_idx ON auth.users (rank) WHERE rank <> '';
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)